lib/apiservers/engine/backends/errors.go

```diff
@@ -47,6 +47,11 @@ func VolumeCreateNotFoundError(msg string) error {
 	return derr.NewErrorWithStatusCode(fmt.Errorf("No volume store named (%s) exists", msg), http.StatusInternalServerError)
 }
 
+// VolumeNotFoundError returns a 404 docker error for a volume create request.
```

Comment:
Maybe actually `... for a volume get request` since this could also be return by `VolumeJoin`